examples/Image Validation/Readme.md

```diff
@@ -0,0 +1,70 @@
+Dropzone for images validating dimensions and type of dropped images
+
+```
+class BasicValidatedImage extends React.Component {
+  constructor() {
+    super()
+    this.state = { files: [] }
+  }
+
+  onDrop(accepted, rejected, dropZoneRef) => {
+    let errors = [];
+
+    var acceptedImg = accepted[0];
+    if (acceptedImg.type !== 'image/jpeg') {
```

Comment:
This should still be `accept` prop since it improves UX — we can show in advance if it will be accepted or rejected.